frontend/src/components/user-dropdown.js

```diff
@@ -48,16 +49,16 @@ const UserDropdown = () => {
       )}
     >
       <DropdownList>
-        <DropdownItem key="profile">
-          <Link to="/profile/user">Profile</Link>
+        <DropdownItem key="profile" onClick={() => navigate('/profile/user')}>
```

Comment:
Alternatively, you could use `onSelect` for the `Dropdown` to navigate based off the value